MultiPage-Clone/css/main.css

```diff
@@ -1,3 +1,212 @@
 /* Add your CSS here */
-
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+@font-face{
+    font-family: Quicksand;
+    src: url(/MultiPage-Clone/fonts/Quicksand/Quicksand-VariableFont_wght.ttf);
+    font-family: poppins;
+    src: url(MultiPage-Clone/fonts/poppins);
+}
 /* Dont' forget to link this file to your HTML in the <head> */
+:root {
+ --clr-Dark-beige:#E6DACD;
+ --clr-Light-beige:#F4ECE6;
+ --clr-Royal-blue:#0150FD;
+ --clr-Black:#000;
+ --clr-White:#FFF;
+}
+nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 150px;
+    background: var(--clr-White);
+    position: fixed; 
+    top: 0; 
+    width: 100%;
+    z-index: 1000; 
+}
+.logo{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    margin-left: 2rem;
+}
+.logo1{
+    display: flex;
+    gap: 10px;
+    justify-content: center;
+    align-items: center;
+}
+.logo1 img{
+    width: 16px;
+    height: 16px;
+}
+.logo1 span:nth-child(3){
```

Comment:
What do you want to do with the :nth-child selectors? 
To have different elements of logo1 to have different font and padding?
Is it easy to write the CSS? Would you understand in a month what you meant?
Can you think of another way to express in CSS what you need?